src/main/java/com/thebois/controllers/toolbar/ToolbarController.java

```diff
@@ -0,0 +1,67 @@
+package com.thebois.controllers.toolbar;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.badlogic.gdx.InputProcessor;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+import com.thebois.controllers.IController;
+import com.thebois.models.world.World;
+import com.thebois.views.IProjector;
+import com.thebois.views.info.IActorView;
+import com.thebois.views.toolbar.ToolbarView;
+
+/**
+ * Container class for controllers that manage the toolbar.
+ */
+public class ToolbarController implements IController<ToolbarView> {
+
+    private final Collection<IController<IActorView>> controllers;
+    private final Collection<InputProcessor> inputProcessors;
+    private final ToolbarView toolbarView;
+
+    /**
+     * Instantiate with all controller and views used for the toolbar.
+     *
+     * @param world     The world that controllers manage.
+     * @param skin      Used to style widgets with.
+     * @param projector Used to translate screen coordinate to world coordinates.
+     */
+    public ToolbarController(
+        final World world, final Skin skin, final IProjector projector) {
+        final PlaceStructureController placeStructureController =
+            new PlaceStructureController(world, projector, skin);
+
+        controllers = List.of(placeStructureController);
+        inputProcessors = List.of(placeStructureController);
+
+        toolbarView = createToolbarView();
+
+        placeStructureController.setToolbarWidget(toolbarView.getPane());
+    }
+
+    private ToolbarView createToolbarView() {
+        final List<IActorView> views = controllers.stream().map(IController::getView).collect(
+            Collectors.toList());
+        return new ToolbarView(views);
+    }
+
+    @Override
+    public ToolbarView getView() {
+        return toolbarView;
+    }
```

Comment:
Wasn't one of the points of having the controllers create the views that the outside world doesn't need to know about the view? They just need the `IView`, right?

Reply:
Gonna quote you 
> Ah, they should all ideally be extending Actor, but that's another refactor so let's just leave it as it is right now.

I could easily make them(ToolbarView and InfoView) in IActorViews, or just leave them like this. Because making them into actors would take som time.

Reply:
Not all of them, but I could make InfoView and ToolbarView into IActorViews

Reply:
Why not `GameView`?

If it can't be done uniformly, then let's not bother.

Reply:
>then let's not bother

Oh, well if it was that easy, then just leave it like you've done now.